plugin/src/test/kotlin/org/jlleitschuh/gradle/ktlint/KtlintPluginTest.kt

```diff
@@ -185,4 +185,15 @@ class KtlintPluginTest : AbstractPluginTest() {
             assertThat(task(":ktlintMainCheck")!!.outcome, equalTo(TaskOutcome.SUCCESS))
         }
     }
+
+    @Test
+    fun `Should generate code style files`() {
+        projectRoot.withCleanSources()
+        val ideaRootDir = projectRoot.resolve(".idea").apply { mkdir() }
+
+        build("ktlintApplyToIDEA").apply {
+            assertThat(task(":ktlintApplyToIdea")!!.outcome, equalTo(TaskOutcome.SUCCESS))
```

Comment:
It would be better to do this, so the test will fail properly (throwing proper assertion framework exception that shows exactly what is wrong, instead of `KotlinNullPointerException`)
`task(":ktlintApplyToIdea")?.outcome`